compiler/damlc/daml-stdlib-src/DA/Internal/Exception.daml

```diff
@@ -121,28 +119,6 @@ instance HasFromAnyException ArithmeticError where
 
 --------------------------------------------------------------
 
-instance HasField "message" ContractError Text where
```

Comment:
We still need this in Daml. We just need to change the instances. Maybe just turn them into `error` calls and then @sofiafaro-da or someone else can fix it up later.